### PR TITLE
Cultrunes no longer work on Supply shuttle.

### PR DESCRIPTION
**Pull request by AliiTea**
Merge this naow.
Fixes: https://github.com/HippieStationCode/HippieStation13/issues/2157
_no one sees the dildos..._

### CODE REVIEW

**Comment by Nokington** [comment]
Nope

**Comment by Crystalwarrior** [comment]
stop being a meme

**Comment by doobiedoo23** [comment]
Trying to sneak shit in with your code is fucking stupid.

**Comment by AliiTea** [comment]
I wasn't trying to sneak shit in holy shit

**Comment by Crystalwarrior** [comment]
> not a meme
> not trying to sneak shit
> then what did you even do

**Comment by AliiTea** [comment]
It was a fuckup when I was actually trying to add in dildos

File: code/controllers/subsystem/shuttles/supply.dm [hunk 1]
```diff
@@ -219,6 +219,7 @@
 		/obj/machinery/teleport/station,
 		/obj/machinery/teleport/hub,
 		/obj/machinery/telepad
```
Comment (Crystalwarrior) [inline code/controllers/subsystem/shuttles/supply.dm]: You missed a , here.

**Comment by AliiTea** [comment]
Suck a cock @Crystalwarrior 

fucking commas are shit

**Comment by Pyko1** [comment]
And this, is why we need Unapproved Memes label!

**Comment by Spacedong** [comment]
how about we just completely deport nexendia this time

**Comment by KayeArray** [comment]
gib dildos plox

**Comment by Carbonhell** [comment]
Bugfix, code is ok,merging.

**Pull request merged**